### ISSUE TITLE
Start fails on Linux  - electron.cmd: command not found

### ISSUE THREAD

**Issue by lucamartinetti** [ElectronNET/electron.net-api-demos#1]
On Ubuntu Zesty when I try to start the demo I get the following
```
 $ dotnet electronize start          
Start Electron Desktop Application...
Microsoft (R) Build Engine version 15.3.409.57025 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  ElectronNET-API-Demos -> /home/luca/Github/electron.net-api-demos/ElectronNET-API-Demos/bin/Debug/netcoreapp2.0/win10-x64/ElectronNET-API-Demos.dll
  ElectronNET-API-Demos -> /home/luca/Github/electron.net-api-demos/ElectronNET-API-Demos/obj/Host/bin/


Start npm install...
npm WARN ElectronNET.Host@1.0.0 No description
npm WARN ElectronNET.Host@1.0.0 No repository field.


Invoke electron.cmd - in dir: /home/luca/Github/electron.net-api-demos/ElectronNET-API-Demos/obj/Host/node_modules/.bin
bash: line 1: electron.cmd: command not found
```

**Comment by robertmuehsig** [comment]
This issue is tracked here: https://github.com/ElectronNET/Electron.NET/issues/36 

The root cause is, that the current "start-command" assumes Windows as the OS. With 0.0.6 this should be fixed. I'm closing this here. Sorry for this obvious mistake on our side :)

**Comment by lucamartinetti** [comment]
Thanks!

**Comment by robertmuehsig** [comment]
I just released the 0.0.6 packages, I edited the .csproj to use the 0.0.6 API NuGet package and if you do a dotnet restore the 0.0.6 version of the CLI project should be included as well.

This should hopefully fix the osx/linux start problems.
